packages/ember-toucan-core/src/components/form/fields/input.hbs

```diff
@@ -35,21 +35,37 @@
         {{/if}}
       </field.Hint>
     {{/if}}
-
     <field.Control class="mt-1.5 flex">
       <Form::Controls::Input
         id={{field.id}}
         aria-describedby="{{if @hint field.hintId}} {{if @error field.errorId}}"
         aria-invalid={{if @error "true"}}
+        class="w-full"
         @isDisabled={{@isDisabled}}
         @value={{@value}}
         @onChange={{@onChange}}
         @hasError={{this.hasError}}
         ...attributes
       />
     </field.Control>
-    {{#if @error}}
-      <field.Error id={{field.errorId}} @error={{@error}} data-error />
-    {{/if}}
+
+    <div
+      class="type-xs-tight text-critical mt-1.5 flex justify-between
+        {{if this.hasMoreThanOneError 'items-start' 'items-center'}}"
+    >
+      {{#if @error}}
+        <field.Error
+          class="flex-3"
+          id={{field.errorId}}
+          @error={{@error}}
+          data-error
+        />
+      {{/if}}
+      {{#if (has-block "character")}}
+        <div class="text-body-and-labels flex-1 text-right">
+          {{yield (hash id=field.id) to="character"}}
+        </div>
+      {{/if}}
+    </div>
```

Comment:
I'm wondering if we can prevent another wrapping div by _only_ rendering it if there's a `has-block "character"` line?  Maybe we can leave `error.hbs` as is today and then do something like:


```hbs
{{#if (has-block "character")}}
  <div class="flex items-center">
    {{#if @error}}
      <field.Error
        class="flex-3"
        id={{field.errorId}}
        @error={{@error}}
        data-error
      />
    {{/if}}

    <div class="text-body-and-labels mt-1.5 flex-1 text-right">
      {{yield (hash id=field.id) to="character"}}
    </div>
  </div>
{{else if @error}}
  <field.Error
    {{! NOTE: Not applying flex-3 here since we don't need it }}
    id={{field.errorId}}
    @error={{@error}}
    data-error
  />
{{/if}}
```